File: linked_list_ToBinary/binary.py
class ListNode:
    def __init__(self, val=0, next=None):
        self.val = val
        self.next = next


class Solution:
    def getDecimalValue(self, head: ListNode) -> int:
        binary_str = ""
        current = head

        while current is not None:
            binary_str += str(current.val)
            current = current.next

        return int(binary_str, 2)

File: linked_list_ToBinary/binary.cs
using System;

public class ListNode
{
    public int val;
    public ListNode next;

    public ListNode(int val = 0, ListNode next = null)
    {
        this.val = val;
        this.next = next;
    }
}

public class Solution
{
    public int GetDecimalValue(ListNode head)
    {
        string binaryStr = "";
        while (head != null)
        {
            binaryStr += head.val.ToString();
            head = head.next;
        }
        return Convert.ToInt32(binaryStr, 2);
    }
}
